### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [front-matter/invenio-rdm-starter#12] [General]
# index

https://starter.front-matter.io/

**Comment by fc47011** [comment]
Hi,

Thank you. The setup is quiter easy. But Im having an error "Internal Server Error" when connecting to https:\\localhost. I can send full logs but here is a sample:

b_1      | 2024-10-08 15:19:49.718 UTC [60] ERROR:  relation "banners" does not exist at character 353
db_1      | 2024-10-08 15:19:49.718 UTC [60] STATEMENT:  SELECT banners.created AS banners_created, banners.updated AS banners_updated, banners.id AS banners_id, banners.message AS banners_message, banurl_path AS banners_url_path, banners.category AS banners_category, banners.start_datetime AS banners_start_datetime, banners.end_datetime AS banners_end_datetime, banners.active AS banners_active
db_1      |     FROM banners
db_1      |     WHERE banners.active IS true AND banners.start_datetime <= '2024-10-08T15:19:49.509173'::timestamp AND (banners.end_datetime IS NULL OR banners.end_datetime >= '2024-10-08T15:19:49.5091timestamp) AND (banners.url_path IS NULL OR ('/' LIKE banners.url_path || '%'))
web_1     | [2024-10-08 15:19:49,721] ERROR in app: Exception on / [GET]
web_1     | Traceback (most recent call last):
web_1     |   File "/opt/invenio/.venv/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 1910, in _execute_context
web_1     |     self.dialect.do_execute(
web_1     |   File "/opt/invenio/.venv/lib/python3.12/site-packages/sqlalchemy/engine/default.py", line 736, in do_execute
web_1     |     cursor.execute(statement, parameters)
web_1     | psycopg2.errors.UndefinedTable: relation "banners" does not exist
web_1     | LINE 2: FROM banners

Thanks,
